frontend/.jshintrc

```diff
@@ -10,7 +10,6 @@
     "angular",
     "inject",
     "module",
-    "protractor",
```

Comment:
express runs the living styleguide. You can export it with `gulp styleguide`, but that won't load all assets.